internal/nix/nixprofile/profile.go

```diff
@@ -133,7 +133,8 @@ func ProfileListIndex(args *ProfileListIndexArgs) (int, error) {
 			return -1, errors.Wrapf(err, "failed to get installable for %s", args.DevPkg.String())
 		}
 		for _, item := range items {
-			if pathInStore == item.nixStorePath {
+			if len(item.nixStorePaths) == 1 && // this should always be true
+				pathInStore == item.nixStorePaths[0] {
```

Comment:
Thisshould be  true, yes. 

Should we assert that by erroring if len of item.nixStorePaths > 1
OR just iterate over the item.nixStorePaths so we don't mind if the assumption is violated and things keep working...